README.md

```diff
@@ -9,6 +9,14 @@ $ source venv/bin/activate
 (venv) $ pip install -r requirements.txt
 ```
 
+To run the app, you should first edit the configuration:
+
+1. Rename the config file from `config.py.dist` to `config.py`.
+2. Change the `SECRET_KEY` to one of your own. Make it unique.
+4. Change `TIMESYNC_URL` to the URL of the TimeSync instance to
+communicate with.
+5. If you want to run it in debug mode, make sure you set `DEBUG` to `True`.
```

Comment:
You should also add the command to run the app:

```
python run.py
```

and explain that it is now running on `http://localhost:5000` and the login page can be accessed at `/login`.